integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/BenchmarkRunner.scala

```diff
@@ -89,23 +89,26 @@ object BenchmarkRunner {
                   path,
                   iterations = conf.iterations(),
                   summaryFilePrefix = summaryFilePrefixWithQuery,
-                  gcBetweenRuns = conf.gcBetweenRuns())
+                  gcBetweenRuns = conf.gcBetweenRuns(),
+                  generateDotGraph = conf.generateDot())
```

Comment:
Not a change that needs to be in this PR, but the number of options is getting to the point where we should consider using a builder pattern, building an options object and passing that.